ntbs-service/deployments/test.yml

```diff
@@ -26,6 +26,11 @@ spec:
               secretKeyRef:
                 name: test-audit-db-connection-string
                 key: connectionString
+          - name: ConnectionStrings__reporting
+          valueFrom:
+            secretKeyRef:
+              name: test-reporting-db-connection-string
+              key: connectionString
```

Comment:
indentation is off here

Reply:
This is odd, it shows file on my local version